### ISSUE TITLE
Abnormal training loss curve for yolov7-tiny

### ISSUE THREAD

**Issue by darkxzk** [AlexeyAB/darknet#8716]
When I trained yolov7-tiny on the dataset, I noticed that the loss curve decreased and then increased a bit before leveling off. I ran the same experiment with Yolov4-Tiny and found no such issues. This seems to be unique to Yolov7-Tiny, anyone know why? Isn't this overfitting?
![chart](https://user-images.githubusercontent.com/119394700/204485181-7fed65b7-1a0b-49e6-ad0e-1073e3278caa.png)

**Comment by darkxzk** [comment]
@AlexeyAB

**Comment by 1027663760** [comment]
[yolo]
objectness_smooth=0

这样设置就好了

**Comment by dlavrantonis** [comment]
can someone please explain what does this objectness parameter do? Indeed the loss curve gets back to normal if do the above but the model itself performed much worse for me. It started having way too many false positives. Has anybody else experienced the same?

**Comment by stephanecharette** [comment]
I think everyone that has commented on YOLOv7 in the Darknet/YOLO discord has had more-or-less the same kinds of things to say about it.  None of us have gotten it to work better than YOLOv4 or YOLOv4-tiny.

**Comment by dlavrantonis** [comment]
@stephanecharette what I'm saying is that despite the abnormal curve when objectness_smooth=1, I'm getting better results compared to when objectness_smooth=0. 

Comparing with yolov4-tiny is a different topic. My results on this is that yolov7-tiny scores 9% higher map@0.5 in crowdhuman dataset - modified to detect full bodies only, not heads(=1 class only). In particular v4 scores 68.27% and v7 77.45%. Unless I'm mistaken somewhere, I cant say that I'm getting worse performance in v7. 

Inference speed is a bit slower thats true. And overall the detection scores aren't as high as the ones observed in v4 but still map -which is the ultimate metric- was better by a non negligible margin.

**Comment by EvanBrownVTM** [comment]
yolov7 is shit
